gitfs/utils/args.py

```diff
@@ -1,3 +1,4 @@
+# Copyright 2015 Justus Perlwitz
```

Comment:
The line added breaks the apache 2.0 license under which the code is published. Please check yourself http://www.apache.org/dev/apply-license.html#contributor-copyright